DESCRIPTION

```diff
@@ -1,19 +1,19 @@
 Type: Package
 Package: sass
-Version: 0.1.2.9000
+Version: 0.1.3
 Title: Syntactically Awesome Style Sheets ('Sass')
 Description: An 'SCSS' compiler, powered by the 'LibSass' library. With this,
     R developers can use variables, inheritance, and functions to generate
     dynamic style sheets. The package uses the 'Sass CSS' extension language,
     which is stable, powerful, and CSS compatible.
 Authors@R: c(
-    person("Joe", "Cheng", , "joe@rstudio.com", c("aut", "cre")),
+    person("Joe", "Cheng", , "joe@rstudio.com", "aut"),
     person("Timothy", "Mastny", , "tim.mastny@gmail.com", "aut"),
     person("Richard", "Iannone", , "rich@rstudio.com", "aut",
        comment = c(ORCID = "0000-0003-3925-190X")),
     person("Barret", "Schloerke", , "barret@rstudio.com", "aut",
            comment = c(ORCID = "0000-0001-9986-114X")),
-    person("Carson", "Sievert", , "carson@rstudio.com", "aut",
+    person("Carson", "Sievert", , "carson@rstudio.com", c("aut", "cre"),
```

Comment:
We will definitely have to remember for me to send the transfer email when we submit